mkt/commonplace/views.py

```diff
@@ -155,7 +82,70 @@ def commonplace(request, repo, **kwargs):
         region_middleware = RegionMiddleware()
         ctx['geoip_region'] = region_middleware.region_from_request(request)
 
-    return render(request, 'commonplace/index.html', ctx)
+    if repo in settings.REACT_REPOS:
+        return render(request, 'commonplace/index_react.html', ctx)
+    else:
```

Comment:
Sorry, I know it's annoying. But we want it to `return None` if, for some reason, `repo` isn't in either of those values, right? If it silently fails in that way but returns the commonplace template, we might miss the subtle bug. A lot easier to realize there's a problem and track it down if it returns `None` rather the wrong template, right?